README.md

```diff
@@ -7,7 +7,7 @@
 </p>
 <p align="center">
   <a href="https://travis-ci.org/jdneo/vscode-leetcode">
-    <img src="https://img.shields.io/travis/jdneo/vscode-leetcode.svg?style=flat-square" alt="">
+    <a href="https://opencollective.com/vscode-leetcode" alt="Financial Contributors on Open Collective"><img src="https://opencollective.com/vscode-leetcode/all/badge.svg?label=financial+contributors" /></a> <img src="https://img.shields.io/travis/jdneo/vscode-leetcode.svg?style=flat-square" alt="">
```

Comment:
Could we use the badge from:`https://shields.io/` and adding the `flat-square` style? I saw it already support `Open Collective`